apis/nucleus/src/components/listbox/components/__tests__/list-box-row-column.spec.jsx

```diff
@@ -834,6 +834,30 @@ describe('<ListBoxRowColumn />', () => {
       await testRenderer.unmount();
     });
 
+    it('should get correct text alignment', async () => {
+      const index = 0;
+      const style = {};
+      const textAlign = 'right';
+      const data = {
+        onMouseDown: sandbox.spy(),
+        onMouseUp: sandbox.spy(),
+        onMouseEnter: sandbox.spy(),
+        onClick: sandbox.spy(),
+        keyboard,
+        pages: [],
+        actions,
+        textAlign,
+      };
+      const testRenderer = await render(
+        <ThemeProvider theme={theme}>
+          <ListBoxRowColumn index={index} style={style} data={data} column={rowCol === 'column'} />
+        </ThemeProvider>
+      );
+      const testInstance = testRenderer.root;
+      const type = testInstance.findByType(Grid);
+      expect(type.props.children[1].props.style.justifyContent).to.equal(textAlign);
```

Comment:
Best to write the expect as a new string `'right'`, because `textAlign` could have changed on the journey through the component (i.e. if it's not immutable).